docs/pages/v3/compound-3.md

```diff
@@ -98,6 +118,26 @@ deployments:
       USDC: '0x4fed3d02D095f7D92AF161311fA6Ef23dc8dA040'
       WAVAX: '0xA2c25E48269e3f89A60b2CC8e02AAfEeB3BAb761'
       WBTC.e: '0xfa78400e01Fc9da830Cb2F13B3e7E18F813414Ff'
+  Polygon Mumbai - USDC Base:
+    tab_text: Mumbai USDC
+    blockscan_origin: 'https://mumbai.polygonscan.com/'
+    contracts:
+      cUSDCv3: '0xF09F0369aB0a875254fB565E52226c88f10Bc839'
+      cUSDCv3 Implementation: '0x6Cd33556D458aC78Cc17b55Eb75B17d23E4deE57'
+      cUSDCv3 Ext: '0x1c3080d7fd5c97A58E0F2EA19e9Eec4745dC4BDe'
+      Configurator: '0x64550801B8bf3BF4D8792d46D8903F82e2EC95A9'
+      Configurator Implementation: '0xA5615c22c3Ab520A77b29F3d32C2831156dFbE7B'
+      Proxy Admin: '0xfE14E3BdCADDe7ccD3c8EeE3CcC11e3FC8184E47'
+      Comet Factory: '0xF09F0369aB0a875254fB565E52226c88f10Bc839'
+      Timelock: '0x90b1f90Ed6477d5Ee1Ff14Bef670266DaE9eEb92'
+      Bridge Receiver: '0xe195d2cBf7f20E40Cf701a9fA3F01fE89bA5a1da'
+      Rewards: '0x0785f2AC0dCBEDEE4b8D62c25A34098E9A0dF4bB'
+      Bulker: '0x990D086E52B132f5b4f769829612F756a4a20bb8'
+      DAI: '0x0FCAa9c899EC5A91eBc3D5Dd869De833b06fB046'
```

Comment:
`0x0FCAa9c899EC5A91eBc3D5Dd869De833b06fB046`

Reply:
The ones you have here are the price feed addresses instead of the token addresses